### ISSUE TITLE
Modeling

### ISSUE THREAD

**Issue by nateschor** [nateschor/OBP#4]
See how the following models perform on the validation data:

- [x] sample mean of training data for each player
- [x] OLS
- [x] LASSO/Ridge
- [x] Non-linear model like splines

Use best model to get performance on test data (determine metric, RMSE, look into others also)

Create and fill in the table below:

| Model  | Training Years | Training RMSE | Validation Years | Validation RMSE | Test Year | Test RMSE
| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
| Player sample mean  |  1975-2017 | NA  | 2018-2020  | .046 | Not evaluated on test data | NA |
| OLS  | 1975-2017  |  .034 | 2018-2020    | .040  | Not evaluated on test data | NA |
| Splines  | 1975-2017  | .035  | 2018-2020| .043  | Not evaluated on test data | NA |
| Splines  | 1975-2019  | NA  |  All non-test data used for training, no validation set  | NA  | 2020 | .050 |

Spline variable groups:

| Group| OBP Lag 1|  OBP Lag 2| OBP Lag 3 | BB Lag 1 | BB Lag 2 | BB Lag 3 |
| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
| 1 | 1|  1 | 0 | 0 | 0 | 0|
| 2 |  1 |  1 | 1| 0 | 0 | 0|
| 3 |  1 |  1 | 0 | 1| 0 | 0|
| 4 |  1 |  1 | 0 | 1| 1| 0|
| 5 |  1 |  1 | 1| 1| 1| 0|
